src/operations/forms/reply/components/ReplyFormBody.jsx

```diff
@@ -0,0 +1,147 @@
+import { useState } from "react";
+import { Button } from "@material-tailwind/react";
+
+const camelize = (str) => {
+  return str.replace(/(?:^\w|[A-Z]|\b\w|\s+)/g, function (match, index) {
+    if (+match === 0) return ""; // or if (/\s+/.test(match)) for white spaces
+    return index === 0 ? match.toLowerCase() : match.toUpperCase();
+  });
+};
+
+export function ReplyFormBody() {
+  const [formContent] = useState([
+    {
+      id: 0,
+      name: "0",
+      label: "what is your name ?",
+      required: false,
+      question_type: "short_answer",
+      list: [],
+    },
+    {
+      id: 1,
+      name: "1",
+      label: "Describe yourself ?",
+      required: false,
+      question_type: "paragraph",
+      list: [],
+    },
+    {
+      id: 3,
+      name: "3",
+      label: "Select your hobbies",
+      required: false,
+      question_type: "checkbox",
+      list: ["Reading", "Gardening", "Cooking"],
+    },
+    {
+      id: 4,
+      name: "4",
+      label: "Select your preferred payment method",
+      required: false,
+      question_type: "radio",
+      list: ["Credit Card", "PayPal", "Bitcoin"],
+    },
+  ]);
+
+  const submitForm = (e) => {
+    e.preventDefault();
+
+    //loop through our questions & get values based on the element name
+    const formTargets = e.target;
+    const data = [];
+    formContent.map((content) => {
+      const element = camelize(content.label);
+      if (content.question_type === "checkbox") {
+        const checkboxes = formTargets[element];
+        const selectedOptions = Array.from(checkboxes)
+          .filter((checkbox) => checkbox.checked)
+          .map((checkbox) => checkbox.value);
+        data.push({
+          question: content.label,
+          answer: selectedOptions,
+        });
+      } else {
+        data.push({
+          question: content.label,
+          answer: formTargets[element].value,
+        });
+      }
+    });
+
+    console.log("form data", data);
+  };
+
+  return (
+    <form onSubmit={submitForm} className="reply-block">
```

Comment:
there will be a bounce of questions with reply so I mean, create state or store or something else to handle them : 

{questionId: "uuid",
answer: ""}